types/angular/jqlite.d.ts

```diff
@@ -651,6 +651,7 @@ interface JQuery {
 
     // Undocumented
     length: number;
+    [index: number]: TElement;
```

Comment:
Actually, maybe consider leaving it open for a day or two and we can bug the TS member doing reviews this week to figure out something.